### ISSUE TITLE
YII_DEBUG and YII_ENV settings ignored

### ISSUE THREAD

**Issue by XzAeRo** [trntv/yii2-starter-kit#407]
**TL;DR**: `YII_DEBUG` and `YII_ENV` are fixed to `false` and `'prod'` respectively and all the changes get ignored.

**Long description**:
Using the latest version with Vagrant. Also tested in Mac OS X and native Ubuntu Server 14.04. All of them with a fresh `composer install` (no previous vendor folder).
The `.env` settings for `YII_DEBUG` and `YII_ENV` are set like this:

```bash
YII_DEBUG = true
YII_ENV = dev
```

But inside the application, Yii2 keeps resolving the settings to `YII_DEBUG => false` and `YII_ENV => 'prod'`.
I even tried directly setting this in `common/config/bootstrap.php`:

```php
defined('YII_DEBUG') or define('YII_DEBUG', true);
defined('YII_ENV') or define('YII_ENV', 'dev');
```

But the application still resolves to `YII_DEBUG => false` and `YII_ENV => 'prod'` (`YII_ENV_PROD => true` and `YII_ENV_DEV => false`).

The rest of the `.env` files gets parsed normally.
This even seems like a Yii2 issue, but I'm not sure, and the latest `env()` function addition in the latest commits made me a little suspicious.

**Comment by ignatenkovnikita** [comment]
I add this environment in frontend/web/index, but this no good idea.

**Comment by XzAeRo** [comment]
I tried adding this to `frontend/web/index` but nothing changed.
This is making me crazy, because I can't get the debug toolbar to appear, and all the errors get masked behind the production environment.

**Comment by notemood** [comment]
Add the following in frontend/web/index.php fixed this for me
```php
defined('YII_DEBUG') or define('YII_DEBUG', true); 
defined('YII_ENV') or define('YII_ENV', 'dev');
```

**Comment by XzAeRo** [comment]
I did, it doesn't make a difference.

**Comment by iadj** [comment]
I had to change the definitions in `Yii.php` in `vendor/yiisoft/yii2` in order to make it work with the latest version of Yii2.

**Comment by trntv** [comment]
fixed, sorry for that
